lib/rubies.rb

```diff
@@ -6,3 +6,4 @@
 require 'rubies/random_hash'
 require 'rubies/random_data_structure'
 require 'rubies/game'
+require 'ripper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.